### ISSUE TITLE
Feature Request: Submit when multiple contact forms on page or post

### ISSUE THREAD

**Issue by azurecurve** [TukuToi/tukutoi-contact-form#10]
I tested a page with two contact forms:
```
[tkt_cntct_frm_contact_form  id='contact-us']

[tkt_cntct_frm_contact_form  id='question']
```
When I submitted one, both forms showed the success message:
```
Thanks for your e-mail! We'll get back to you as soon as we can.
Thanks for your e-mail! We'll get back to you as soon as we can.
```

**Comment by smileBeda** [comment]
Yes, the contact form doesn’t support two forms on the same page, as it’s a contact form and shouldn’t need more than one instance

maybe in future I’ll add support for more than one form on the same page, however for now only one form is supposed to be on each page 
See also #5

**Comment by azurecurve** [comment]
I think I was remembering seeing that issue, but not the resolution and thinking the id had to do with allowing it.

But the id can be used for filtering suffixes and other elements.

**Comment by smileBeda** [comment]
The ID mainly is for the filters and actions so you can target the correct form with the code
Also it is used in the HTML markup of the form

It should make it easier since its there to add support for many forms on the same page, but many forms on the same page mean a lot more. Like wp_nonce needs to be unique, error handling needs to be unique, and so on.
Since Contact forms literally almost never appear more than once on a page (I am pretty sure no Theme's contact form for example can be added more than once) I decided to not support this for now since it diverges from the purpose of a contact form, and plugins should do "what they say they do"

I will keep this here as a Feature Request and implement if/when I have time for it.
